_drafts/dei-coop.md

```diff
@@ -6,17 +6,17 @@ categories:
+In the past week we have seen protests break out here in Seattle, and across the country in response to the killing of George Floyd, a Black man who was killed by the Minneapolis police.  All too often STEM spaces tend to remain quiet on issues of racial injustice leaning into the idea that the field of STEM itself is unbiased and therefore these issues are not relevant to our work. However, this is untrue. By now many people have heard the story of Henrietta Lacks, a Black woman who was diagnosed and treated for cervical cancer at Johns Hopkins in the 1950s. Her tissue samples were shared and propagated without her knowledge or permission and are still used in research today, while her family has seen no compensation. We've seen the rise of machine learning algorithms that further perpetuate racist biases in the decisions they make. We've read the peer reviewed articles and first-hand accounts detailing racisim in academic and scientific institutions.
 
-Tech and data science spaces tend to be overwhelmingly white and this is true here at Fred Hutch, and within The Coop. I am writing this post to express The Coop's firm stance against racial injustice in our community and beyond. We are committed to uplifting and amplifying the voices of Black biologists and data scientists. We are committed to incorporating discussions of ethics into our classes on data science. We are committed to hearing from our community when mistakes are inevitably made, or when simply not enough is being done. If you have thoughts, questions, or concerns please feel empowered and encouraged to reach out to either Kate Hertwick or myself (Lauren Wolfe) through Slack, Teams, or email.
+Tech and data science spaces tend to be overwhelmingly white. This is true here at Fred Hutch and within The Coop. I am writing this post to express The Coop's firm stance against racial injustice in our community and beyond. We are committed to uplifting and amplifying the voices of Black biologists and data scientists. We are committed to incorporating discussions of ethics into our classes on data science. We are committed to hearing from our community when we make mistakes, or when simply not enough is being done. If you have thoughts, questions, or concerns please feel empowered and encouraged to reach out to either Kate Hertwick or myself (Lauren Wolfe) through Slack, Teams, or email.
 
 ## Get involved at Fred Hutch
 
-Fred Hutch's Office of Diversity, Equity, and Inclusion (ODEI) is holding sessions to engage the Fred Hutch commuity in structured conversation about racisim.
+Fred Hutch's Office of Diversity, Equity, and Inclusion (ODEI) is holding sessions to engage the Fred Hutch commuity in structured conversation about racisim. You can register for either of the following sessions by emailing the ODEI at `diversity@fredhutch.org` and requesting a registration link.
```

Comment:
email as `diversity` at `fredhutch.org` (this is how we've included things on the coop website, I think)